pkg/ci/plugins/helm/templates/apply.md

```
@@ -14,6 +14,37 @@
+{{- with .Lifecycle }}
+
+### Release lifecycle
+
+{{- if eq (index . "reason") "external_target" }}
+
+| Field | Value |
+| --- | --- |
+| Applied | `false` |
+| Target kind | `{{ index . "target_kind" }}` |
+| Reason | `external_target` |
+{{- else }}
```

markdownlint-cli2 (0.23.1)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: no_leading_or_trailing; Missing leading pipe

(MD055, table-pipe-style)